### PR TITLE
Use PHP attributes in tests for PHP8+

### PR DESCRIPTION
**Pull request by norkunas**
# Pull Request

## Related issue
Fixes #<issue_number>

## What does this PR do?
- Configures PHP attributes to be used in tests for PHP8+

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: tests/config/config_php7.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+framework:
+    test: true
+    secret: 67d829bf61dc5f87a73fd814e2c9f629
```
Comment (brunoocasali) [inline tests/config/config_php7.yaml:3]: Is this value intentional?

**Review by brunoocasali — COMMENTED** [review]
Just to make sure I get it right... After PHP 8 doctrine supports the `#[...]` notation in the PHP classes instead of making comments?

This will be the default soon?

File: tests/config/config_php7.yaml (hunk 1 above)
Comment (norkunas) [reply tests/config/config_php7.yaml:3]: I left it as it was :)

**Comment by norkunas** [comment]
> Just to make sure I get it right... After PHP 8 doctrine supports the `#[...]` notation in the PHP classes instead of making comments?
> 
> This will be the default soon?

Not sure when the support for annotations will be dropped, but when this bundle will drop support for php 7.4 then the annotations from tests could be removed because I think most people who use php8 are already migrated to attributes or easily can migrate with rector. But those who don't then they will rely on annotations support in ORM itself

**Review by brunoocasali — APPROVED** [review]
Thanks @norkunas for your contribution 🎉  

bors merge

**Pull request merged**